docs/src/main/asciidoc/doc-contribute-docs-howto.adoc

```diff
@@ -33,51 +33,103 @@ We suggest you have the following materials nearby:
 
 To ensure that your content shows up correctly on the link:https://quarkus.io/guides/[Quarkus documentation home page], use the following steps:
 
-. Decide on a content type that best fits the content that you are contributing.
+. Decide on a [diataxis](https://quarkus.io/guides/doc-concept) content type that best fits the content that you are contributing.
```

Comment:
```suggestion
. Decide on a xref:doc-concept.adoc[Diataxis] content type that best fits the content that you are contributing.
```

Reply:
@rolfedh - Apologies. I was in markdown mode instead of Asciidoc when I made the original comment!

Reply:
Lol. Nice catch. I was in copy/paste/don't-think-about-it mode. Thanks for your thorough peer review!